lib/licensee/project_files/license_file.rb

```diff
@@ -20,6 +20,11 @@ def self.name_score(filename)
         return 0.5 if filename =~ /licen[sc]e/i
         0.0
       end
+
+      # case-insensitive block to determine if the given file is LICENSE.lesser
+      def self.lesser_gpl_score(filename)
+        filename.casecmp('license.lesser').zero? ? 1 : 0
```

Comment:
Will this miss COPYING.LESSER? I suspect that's common, given tradition adhered to by many _GPL'd projects of using COPYING rather than LICENSE.